.plop/templates/reactComponent.tsx.hbs

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+{{#if componentHasStyles}}
+import { Wrapper } from "./styles";
+{{else}}
```

Comment:
In my opinion better is: `import * as S from "./styles"`.
When you use it in component it's clear which part is styled and it's easier to name styled components.

Reply:
Good idea, done!